_posts/2017-05-15-rust-at-two-years.md

```diff
@@ -0,0 +1,356 @@
+---
+layout: post
+title: "Two years of Rust"
+author: Carol (Nichols || Goulding)
+description: "Rust, two years after 1.0"
+---
+
+It's been two years since Rust 1.0 was released. Happy second birthday, Rust!
+
+![Group picture from RustFest Berlin][group-pic]{:class="center"}
+
+*Rustaceans at RustFest Berlin, September 2016. Picture by Fiona Castiñeira*
+
+[group-pic]: /images/2017-05-Second-Birthday/rustfest-berlin.jpeg
+
+Rust, a systems programming language focused on safety, speed, and concurrency,
```

Comment:
Yeah I was trying to find where you were talking about... productive fearless systems programming? What was it?